src/main/java/Config/Config.java

```diff
@@ -0,0 +1,69 @@
+package Config;
+
+import Config.Routes.Route;
+import Config.Routes.RouteFactory;
+import Config.Routes.SimpleRouteFactory;
+import Controller.Handler.GetEchoHandle;
+import Controller.Handler.GetHandle;
+import Controller.Handler.Handler;
+import Controller.Handler.PostEchoHandle;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketException;
+import java.util.ArrayList;
+
+import static Config.Method.GET;
+import static Config.Method.POST;
+
+import static Validator.Validator.validArgsLength;
+import static Validator.Validator.validFlag;
+
+public class Config {
+  public static ArrayList<Route> routes = new ArrayList<>();
+  public static ArrayList<Handler> handlers = new ArrayList<>();
+
+  public static void createRoutes() {
+    SimpleRouteFactory factory = new SimpleRouteFactory();
+    RouteFactory routeFactory = new RouteFactory(factory);
+    routes.add(routeFactory.createRoute(GET, "/", new GetHandle()));
+    routes.add(routeFactory.createRoute(GET, "/echo", new GetEchoHandle()));
+    routes.add(routeFactory.createRoute(POST, "/echo", new PostEchoHandle()));
+  }
+
+  public static ArrayList<Handler> allHandlers() {
+    handlers.add(new GetHandle());
+    handlers.add(new GetEchoHandle());
+    handlers.add(new PostEchoHandle());
+    return handlers;
+  }
+
+  public static ArrayList<Route> allRoutes() {
+    return routes;
```

Comment:
To follow up on the above mention of collection objects, and the fact that these objects can be mutated, if you're sharing this list of routes with other objects in the code, then you could potentially be handing around things that change depending on when you run them.  That can lead to some really insidious, hard to debug scenarios.

Maybe either return a copy of the list, or consider wrapping the list in a domain object (`RoutesList` or something).